WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -2310,7 +2310,7 @@ public static void Initialize(SkillTree skillTree, ItemAttributes itemAttrs)
 
             CoreAttributes();
 
-            Implicit = new AttributeSet(skillTree.ImplicitAttributes(Global));
+            Implicit = new AttributeSet(SkillTree.ImplicitAttributes(Global, skillTree.Level));
```

Comment:
Yes, you can check level bounds in SkillTree class, but still have to 
set Compute.Level property in Compute.Initialize method.

It looks silly to have code like this, it's matter of code consistency:
Compute::Initialize() {
...
Level = skilltree._level; // This is single point of Level intialization 
for all calculations within Compute class (no idea what is difference 
now between skillTree._level and SkillTree.Level).
...
skillTree.ImplicitAttributes(Global, skilltree.Level); // Here should be 
passed own Level property (no point reference external property twice, 
once own property has been initialized earlier).
...
}

Well, after second look it's even more mysterious why SkillTree method 
call has own property passed to :)
Is it really necessary to pass skilltree.Level to own method 
(ImplicitAttributes) of skillTree instance? :)